tests/integration_test.rs

```diff
@@ -89,78 +89,16 @@ fn escaped_comment_indicator_in_section_name() {
 
 #[test]
 fn removes_bom() {
-	compare("\u{feff}a=b", "a=b\n");
+	let utf8_bom: [u8; 3] = [0xef, 0xbb, 0xbf];
```

Comment:
See, @florianb? I told you it was a UTF8 BOM. I'm very confused how `\u{feff}` matches this BOM, but it clearly does!